android/library/src/main/java/com/liferay/mobile/screens/ddl/model/Field.java

```diff
@@ -358,32 +408,48 @@ public static DataType valueOf(Element element) {
 		}
 
 		public Field createField(Map<String, Object> attributes, Locale locale, Locale defaultLocale) {
+
+			return createField(attributes, locale, defaultLocale, false);
+		}
+
+		public Field createField(Map<String, Object> attributes, Locale locale, Locale defaultLocale,
+			boolean repeatedField) {
+
+			Field field = null;
+
 			if (STRING.equals(this)) {
 				EditorType editor = EditorType.valueOf(attributes);
 
 				if (editor == EditorType.SELECT || editor == EditorType.RADIO) {
-					return new StringWithOptionsField(attributes, locale, defaultLocale);
+					field = new SelectableOptionsField(attributes, locale, defaultLocale);
```

Comment:
why change the return?

Reply:
We prefer to use multiple returns only for guard statements or loop exits, to increase readability (this method is very big). But it's just a personal preference, we can change it.